io/plugins/eu.esdihumboldt.hale.io.csv.test/src/eu/esdihumboldt/hale/io/csv/CSVInstanceReaderTest.java

```diff
@@ -104,6 +113,32 @@ public void testReadSimple() throws Exception {
 
 	}
 
+	/**
+	 * Test - read a sample csv schema and data where skip is a boolean. It
+	 * simulates the previous version where the first line was either skipped
+	 * (true) or not (false)
+	 * 
+	 * @throws Exception , if an error occurs
+	 */
+	@Test
+	public void testBackwardCompatibilityRead() throws Exception {
+		String typeName = "location";
+		// read Schema ###
+		Schema schema = readCSVSchema("/data/test1.csv", typeName,
+				"java.lang.String,java.lang.String,java.lang.String,java.lang.String",
+				"Name,Xcoord,Ycoord,id", null, null, null);
+
+		// test backward compatibility with skip first line as boolean
+		int numberOfLines = 3; // no. of lines = header + 2 instances
+		InstanceCollection instances_no_skip = readCSVInstances("/data/test1.csv", typeName, false,
```

Comment:
It would be great if you could follow the java naming standards with came case `instancesNoSkip` for `instances_no_skip`. This will keep the standards same across the application.

Does `no` in the variable name represents `NO` or the `number`? Maybe then also update the name of the variable accordingly.